modules/dynamic-plugin-localization.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/overview-dynamic-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="dynamic-plugin-api_{context}"]
+= Translating messages with react-i18next
+
+The link:https://github.com/openshift/console-plugin-template[plugin template] demonstrates how you can translate messages with link:https://www.i18next.com/[react-i18next]. The `v1alpha1` `apiVersion` of `ConsolePlugin` supports `console.openshift.io/use-i18n` annotation.
+
+.Prerequisites
+* You must have the plugin template cloned locally.
+* You must be running the {product-title} web console in a container. You can use either Docker or Podman 3.2.0+.
```

Comment:
This is necessary to test your plugin locally, but isn't strictly required for the steps in this section.